x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/StoreKeyConfig.java

```diff
@@ -78,6 +78,18 @@ X509ExtendedKeyManager createKeyManager(@Nullable Environment environment) {
         Path ksPath = keyStorePath == null ? null : CertParsingUtils.resolvePath(keyStorePath, environment);
```

Comment:
We definitely need a test for this behavior. One in StoreKeyConfigTests that checks that the created KeyManager has the properties we want it to have

Reply:
Added tests